modules/support-reviewing-an-access-request-from-an-email-notification.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-support.adoc
+// * support/getting-support.adoc
+// * service_mesh/v2x/ossm-troubleshooting-istio.adoc
+// * osd_architecture/osd-support.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="support-reviewing-an-access-request-from-an-email-notification_{context}"]
+= Reviewing an access request from an email notification
+
+.Prerequisites
+
+ifndef::openshift-rosa,openshift-dedicated[]
```

Comment:
Same comment about this conditional. Right now it's showing only if the distro is NOT ROSA or dedicated. Not sure if that's what was intended or not, but wondering if it should even be here if it's not being included in anything other than ROSA.

And also to put the prereqs heading inside the conditional to be safe